src/main/resources/reference.conf

```diff
@@ -511,6 +511,41 @@ resourceTypes = {
     }
     reuseIds = true
   }
+  spend-profile = {
+    actionPatternObjects = {
+      "create" = {
+        description = "Can create profiles"
+      }
+      "update" = {
+        description = "Can update profiles"
+      }
+      "delete" = {
+        description = "Can delete profiles"
+      }
+      "list" = {
```

Comment:
all users can list resources they have access to so `list` is not requried

Reply:
We have a similar action for other TDR objects: `read_dataset` action for seeing the dataset definition.
We use the action check on the resource id when accessing a single dataset with this:
`/api/resources/v1/{resourceTypeName}/{resourceId}/action/{action}`

When enumerating, we get all accessible resources using this:
`/api/resources/v1/{resourceTypeName}`

So if I understand, you would recommend using `/api/resources/v1/{resourceTypeName}/{resourceId}/actions` for checking if a user has some access to the object rather than making an action. Yes?